reversion_compare_tests/models.py

```diff
@@ -104,7 +105,7 @@ class VariantModel(models.Model):
 
     TEST_CHOICES = (("a", "alpha"), ("b", "bravo"))
     boolean = models.BooleanField(default=True)
-    null_boolean = models.NullBooleanField()
+    null_boolean = models.BooleanField(null=True) if DJANGO_VERSION[0] >= 3 else models.NullBooleanField()
```

Comment:
@jedie FYI this is unrelated to the diff changes, but it fixes a deprecation warning that was causing a test failure.